### PR TITLE
PMM-7181 Dashboards Add panel with pt--summary information

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1513

### CODE REVIEW

File: dashboards/Node_Summary.json [hunk 1]
```diff
@@ -1001,30 +1001,241 @@
...
+                            "$$hashKey": "object:326",
+                            "color": "#8F3BB8",
+                            "state": 0,
+                            "value": 0
+                        }
+                    ],
+                    "unitFormat": "short"
+                },
+                {
+                    "$$hashKey": "object:292",
```
Comment (atymchuk) [inline dashboards/Node_Summary.json:1144]: $$hashKey ))

File: dashboards/Node_Summary.json [hunk 2]
```diff
@@ -1001,30 +1001,241 @@
...
+                            "$$hashKey": "object:372",
+                            "color": "#CA95E5",
+                            "state": 0,
+                            "value": 0
+                        }
+                    ],
+                    "unitFormat": "short"
+                },
+                {
+                    "$$hashKey": "object:539",
```
Comment (atymchuk) [inline dashboards/Node_Summary.json:1173]: Bunch of hashKeys, let's get rid of them ))

File: dashboards/Node_Summary.json [hunk 3]
```diff
@@ -1001,30 +1001,241 @@
...
+                    "metricName": "/.*   /",
+                    "newTabEnabled": true,
+                    "operatorName": "current",
+                    "prefix": "MongoDB - ",
+                    "sanitizeURLEnabled": false,
+                    "scaledDecimals": null,
+                    "suffix": "",
+                    "thresholds": [
+                        {
+                            "$$hashKey": "object:326",
```
Comment (atymchuk) [inline dashboards/Node_Summary.json:1135]: One more hashKey ))

File: dashboards/Node_Summary.json [hunk 4]
```diff
@@ -1001,30 +1001,241 @@
...
+                    "metricName": "/.*  /",
+                    "newTabEnabled": true,
+                    "operatorName": "current",
+                    "prefix": "PostgreSQL - ",
+                    "sanitizeURLEnabled": false,
+                    "scaledDecimals": null,
+                    "suffix": "",
+                    "thresholds": [
+                        {
+                            "$$hashKey": "object:372",
```
Comment (atymchuk) [inline dashboards/Node_Summary.json:1164]: hash? No hashKey

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: dashboards/Node_Summary.json (hunk 2 above)
Comment (adivinho) [reply dashboards/Node_Summary.json]: done

**Review by atymchuk — DISMISSED** [review]

**Review by lunaticusgreen — APPROVED** [review]
👍

**Pull request merged**